src/ecs/resource.rs

```diff
@@ -24,14 +24,3 @@ use media::Key;
 use std::collections::HashSet;
 
 pub struct PressedKeys(pub HashSet<Key>);
-
-pub struct CameraPosition {
```

Comment:
We'll need to keep the CameraPosition struct if for no other reason than to distinctly identify it in the entity component system's resources. We could potentially convert it to a tuple struct:

pub struct CameraPosition(Vector2<f32>);